<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hostDialog</class>
 <widget class="QDialog" name="hostDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2279</width>
    <height>827</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>2279</width>
    <height>820</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Host</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/icon/apple-touch-icon.png</normaloff>:/icon/apple-touch-icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>2255</width>
     <height>802</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout1" stretch="1,1,1,1,1,1,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>box</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>l dn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="leftDownLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>r dn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rightDownLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>l up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="leftUpLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>r up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rightUpLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>eye box</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>l dn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eyeLeftDownLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_28" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>r dn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eyeRightDownLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>l up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eyeLeftUpLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_30" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>r up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eyeRightUpLabel">
          <property name="text">
           <string>(,)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="eyeOriButton">
        <property name="text">
         <string>Origianl</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eyeResetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>front - blue</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>HueL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hueLSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>96</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>HueH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hueHSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>121</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>SatL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="satLSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>231</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>SatH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="satHSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>ValL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="valLSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>ValH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="valHSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>135</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>back - red</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="hueLLabel2">
          <property name="text">
           <string>HueL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hueL2SpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>HueH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hueH2SpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>SatL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="satL2SpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>170</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>SatH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="satH2SpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>232</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>ValL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="valL2SpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>ValH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="valH2SpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>170</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>800</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_23"/>
    </item>
    <item>
     <widget class="QLabel" name="eyeVideoLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>800</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>arg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="angleLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>toX</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="desXLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>toY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="desYLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>me</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ipMeLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>car</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ipCarLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>eye</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ipEyeLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="ipCarLineEdit">
        <property name="text">
         <string>192.168.1.103</string>
        </property>
        <property name="placeholderText">
         <string>car ip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipEyeLineEdit">
        <property name="text">
         <string>192.168.1.106</string>
        </property>
        <property name="placeholderText">
         <string>eye ip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoStartButton">
        <property name="text">
         <string>Video Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoEndButton">
        <property name="text">
         <string>Video End</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
